caracal/schema/selfcal_schema.yml

```diff
@@ -164,7 +164,7 @@ mapping:
         required: false
         example: '0, 5, 10, 20'
       img_nrdeconvsubimg:
-        desc: Speed-up deconvolution by splitting the image into a number "img_nrdeconvsubimg" of subimages, which are deconvolved in parallel. If set to 1 no parallel deconvolution is performed. If set to 0 the number of subimages is the same as the number of CPUs used by the selfcal worker (see "ncpu" parameter above).
+        desc: Speed-up deconvolution by splitting the image into a number of subimages, which are deconvolved in parallel. This parameter sets the number of subimages as follows. If set to 1 no parallel deconvolution is performed. If set to 0 the number of subimages is the same as the number of CPUs used by the selfcal worker (see "ncpu" parameter above). If set to a number > 1 , the number of subimages is the one requested by the user.
```

Comment:
Sorry to be a pest, but we should reword the final line as: "......... If set to a number > 1 , the number of subimages is _**greater than or equal to**_ the one requested by the user.", since the parameter parallel-deconvolution only gives the maximum size of subimages. I get 15 subimages for the test I am running even when this parameter is set to 10.